CPAC/registration/registration.py

```diff
@@ -3218,14 +3219,20 @@ def coregistration_prep_mean(wf, cfg, strat_pool, pipe_num, opt=None):
         "input",
     ],
     option_val="fmriprep_reference",
-    inputs=["desc-ref_bold"],
-    outputs=["sbref"],
+    inputs=[
+        ("motion-basefile", "desc-preproc_bold"),
+        "FSL-AFNI-bold-ref",
+        "FSL-AFNI-brain-mask",
+        "FSL-AFNI-brain-probseg",
+    ],
+    outputs=["sbref", "desc-unifized_bold", "fMRIprep_brain_mask"],
```

Comment:
Why is this writing out `desc-unifized_bold` and `fMRIprep_brain_mask` to the outputs?
We only need it to write out `sbref`.

Reply:
The idea was to provide an abstraction of that subworkflow - the FSL-AFNI bold mask nodeblock will output `desc-unifized_bold` and the bold mask, and this coregistration prep block will only write out `sbref`, thus maintaining the interfaces of those nodeblock types.